roles/nf-core-biocontainer-temp/templates/site.config.j2

```diff
@@ -0,0 +1,4 @@
+{% if item.0.name=='rnaseq' %}
```

Comment:
Could you add site to this if-statement as so that seq_center can be set to Uppsala for us?

Reply:
Done!